src/main/java/org/sefglobal/scholarx/model/Mentor.java

```diff
@@ -16,9 +18,75 @@
     public Mentor() {
     }
 
+    @Column
```

Comment:
```suggestion
    @Enumerated(EnumType.STRING)
    @Column(nullable = false)
```

Reply:
@Piumal1999 Made the corresponding changes in the code. Also updated a screenshot in the PR iteself.

Reply:
Also are there any fields that should be not null? Or is this fine?

Reply:
I think it's fine